frame/dapps-staking/src/lib.rs

```diff
@@ -502,6 +506,51 @@ impl Default for RewardDestination {
     }
 }
 
+#[derive(Encode, Decode, Clone, Copy, Default, PartialEq, Eq, RuntimeDebug, TypeInfo)]
+pub enum EmbededDestination<AccountId, Balance: AtLeast32BitUnsigned + Default + Copy> {
+    #[default]
+    None,
```

Comment:
I guess we should handle `None` via an `Option<T>`, right?

Reply:
Hmm... Yes.